pkg/config/polling_manager.go

```diff
@@ -50,13 +50,14 @@ var cmLogger = logging.GetLogger("PollingConfigManager")
 // PollingProjectConfigManager maintains a dynamic copy of the project config by continuously polling for the datafile
 // from the Optimizely CDN at a given (configurable) interval.
 type PollingProjectConfigManager struct {
-	datafileURLTemplate string
-	initDatafile        []byte
-	lastModified        string
-	notificationCenter  notification.Center
-	pollingInterval     time.Duration
-	requester           utils.Requester
-	sdkKey              string
+	datafileURLTemplate         string
+	initDatafile                []byte
+	lastModified                string
+	notificationCenter          notification.Center
+	pollingInterval             time.Duration
+	requester                   utils.Requester
+	sdkKey                      string
+	projectConfigUpdateHandlers []func(notification.ProjectConfigUpdateNotification)
```

Comment:
@msohailhussain I mentioned to you offline how we should not pull in the handlers here, but instead externally call `OnProjectConfigUpdate` in a loop and attach the handlers that way.

Reply:
yes Mike, am working on it.